### PR TITLE
[Jupyter] Add RBAC

### PR DESCRIPTION
**Pull request by eliyahu77**

### CODE REVIEW

File: charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{{- if .Values.jupyterNotebook.enabled -}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: juputer-binding
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.jupyterNotebook.serviceAccount }}
```
Comment (alonmr) [inline charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml:8]:
```suggestion
    name: {{ .Values.jupyterNotebook.serviceAccount.name }}
```

File: charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+roleRef:
+  kind: Role
+  name: juputer-role
```
Comment (alonmr) [inline charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml:11]:
```suggestion
  name: jupyter-role
```

File: charts/mlrun-ce/templates/jupyter-notebook/serviceaccount.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{{- if .Values.jupyterNotebook.serviceAccount.create -}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+  name: mlrun-jupyter
```
Comment (alonmr) [inline charts/mlrun-ce/templates/jupyter-notebook/serviceaccount.yaml:6]:
```suggestion
  name: {{ .Values.jupyterNotebook.serviceAccount.name }}
```

File: charts/mlrun-ce/templates/jupyter-notebook/serviceaccount.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{{- if .Values.jupyterNotebook.serviceAccount.create -}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
```
Comment (alonmr) [inline charts/mlrun-ce/templates/jupyter-notebook/serviceaccount.yaml:5]:
```suggestion
```

File: charts/mlrun-ce/values.yaml [hunk 5]
```diff
@@ -147,7 +147,10 @@ mlrun:
     mode: full
 
 jupyterNotebook:
-  awsInstall: false
+  serviceAccount:
+    create: true
```
Comment (alonmr) [inline charts/mlrun-ce/values.yaml:151]: why do you need the create?

**Review by alonmr — COMMENTED** [review]
A few comments

File: charts/mlrun-ce/values.yaml (hunk 5 above)
Comment (eliyahu77) [reply charts/mlrun-ce/values.yaml:151]: in oreder ro be able run kubectl from pod

File: charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{{- if .Values.jupyterNotebook.enabled -}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: jupyter-role-binding
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.jupyterNotebook.serviceAccount.name }}
```
Comment (theSaarco) [inline charts/mlrun-ce/templates/jupyter-notebook/role-binding.yaml:8]: What if `.Values.jupyterNotebook.serviceAccount.name` is empty? The user may ask to `create` but not provide a name (by mistake or not). Should we use a default name here in that case?

**Review by theSaarco — COMMENTED** [review]
I think there's a change needed, see comments.

**Pull request merged**